src/components/widgets-settings/attach-kpi/attach-kpi.component.coffee

```diff
@@ -0,0 +1,95 @@
+module = angular.module('impac.components.widgets-settings.attach-kpis', [])
```

Comment:
I would rename to something like `chart-threshold` as this is really related to attaching a threshold kpi to a chart, and not a generic kpi to a widget